.agents/skills/dynamo-router-starter/scripts/check_router_health.py

```
@@ -0,0 +1,112 @@
+    parser.add_argument("--max-tokens", type=int, default=32)
+    parser.add_argument("--skip-chat", action="store_true")
+    parser.add_argument("--retries", type=int, default=5)
+    parser.add_argument("--retry-sleep", type=float, default=2.0)
+    args = parser.parse_args()
+
+    base_url = args.base_url.rstrip("/")
+    result: dict[str, Any] = {"base_url": base_url, "ok": False, "checks": []}
+
+    models_status = None
+    models_body = None
+    for attempt in range(1, args.retries + 1):
```

Ruff (0.15.13)

[warning] Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)